modules/oe_theme_content_consultation/config/install/core.date_format.oe_consultation_date.yml

```diff
@@ -0,0 +1,7 @@
+langcode: en
+status: true
+dependencies: {  }
+id: oe_consultation_date
+label: 'Consultation date'
+locked: false
+pattern: 'j F Y'
```

Comment:
```suggestion
pattern: 'd F Y'
```

Reply:
The date follows the pattern in the requirements. Why is it wrong as it is?